### ISSUE TITLE
FileChooser does not listen to file filter when in Mode.OPEN

### ISSUE THREAD

**Issue by StrongJoshua** [kotcrab/VisEditor#6]
Like the title says, when the file chooser is in the OPEN mode, then it does not listen to the FileFilter, while it does, when it is in SAVE mode. I'm not sure if this is supposed to be a feature, but it feels like a bug to me.

**Comment by kotcrab** [comment]
I can't reproduce that, file filter is working in both OPEN and SAVE mode however I found issue that setting new file filter does not rebuild file list.

```java
FileChooser chooser = new FileChooser(Mode.OPEN);

chooser.setFileFilter(new FileFilter() {
	@Override
	public boolean accept (File pathname) {
		return false;
	}
});

//here chooser will show all files because file list wasn't rebuilt after settings filter
getStage().addActor(chooser.fadeIn());
```

Maybe this was your issue? Please try navigating to some other directory after setting filter and see if it works.

**Comment by StrongJoshua** [comment]
Yes, navigating to a different directory and back does hide the invalid files. Since I didn't see the file filter hiding the invalid files, I assumed it would instead show an error message when choosing an invalid file, which it didn't.

**Comment by kotcrab** [comment]
Okay, so the actual issue was not rebuilding file list after setting filter, fixed in commit 440fb660e3d5c07f8bdd342cb9fe839c0de0f22b
